werkzeug/exceptions.py

```diff
@@ -672,7 +672,27 @@ def __call__(self, code, *args, **kwargs):
             raise LookupError('no exception for %r' % code)
         raise self.mapping[code](*args, **kwargs)
 
-abort = Aborter()
+
+def abort(status, *args, **kwargs):
```

Comment:
While technically you're right that the first argument `code` used to work and that this change actually breaks code, the documentation used to document it as `status` so that's what I chose. It wasn't a conscious decision, I would've probably named it `code` if I paid closer attention.

However, changing it back now would be even worse (regardless of whether in a bugfix release or a major one)